internal/calc/calculate.go

```diff
@@ -0,0 +1,96 @@
+package calc
+
+import (
+	"errors"
+	"log"
+	"strconv"
+
+	rom "github.com/brandenc40/romannumeral"
+)
+
+// Data type to specify the type of number
+type digits int
+
+// Constants to specify the type of number
+const (
+	_ = iota
+	ARABIC_DIGITS
+	ROMAN_DIGITS
+)
```

Comment:
Более подходящее название для этого типа данных - NumeralSystem. Обычно типы, представляющие собой перечисление, объявляют публичными.

Ты создал специальный тип для обозначения систем исчисления, но не использовал его для типизации элементов перечисления. Также если ты хочешь пропустить одно значение оператора iota, можно сделать просто `iota + 1`, объявлять пустую переменную не нужно. Также `iota` можно использовать для других операций. Например, если тебе нужно перечисление со степенями двойки, можно сделать так: `2 ** iota`. Или использовать побитовые сдвиги: `1 << iota` и т.д.

Стоило` сделать так:
```suggestion
type NumeralSystem int

const (
	ARABIC_DIGITS NumeralSystem = iota
	ROMAN_DIGITS
)
```